app/server.js

```diff
@@ -8,6 +8,222 @@ var uid = require('rand-token').uid;
 
 const PORT = 3001;
 
-http.createServer(function (request, response) {
+//api key storage
+const VALID_API_KEYS = ["98738203-8dsj-0983-9f72-di29dk38djk3", "8di3k99l-5305-8dk3-1849-zjdf938jfj98"];
```

Comment:
These should be held in .env files. That keeps them safe and secret so no one can steal them from Github. Not a big deal here, but worth learning how to do. Just google it!